hadron-wai/src/Hadron/Wai/Request.hs

```diff
@@ -93,13 +93,12 @@ fromHTTPRequest_1_1 (HTTPRequestV1_1 m t h b) =
     }
   where
     buildBody NoRequestBody =
-      let body = pure ""
-          bodyLen = W.KnownLength 0 in
-      (body, bodyLen)
-    buildBody (RequestBody bs) =
-      let body = pure bs
-          bodyLen = W.KnownLength . fromIntegral $ BS.length bs in
-      (body, bodyLen)
+      pure $ pure ""
+    buildBody (RequestBody bs) = do
```

Comment:
I had to get Charles to explain this to me, since I don't know the Wai API. A type signature and comment on this function would be helpful, so it's clear that it's `IO IO`.